ivanti_security_controls/plugin.spec.yaml

```diff
@@ -0,0 +1,223 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: ivanti_security_controls
+title: Ivanti Security Controls
+description: Ivanti Security Controls is a unified IT management platform used for managing and protecting through Patch Management, Application Control, and Asset Inventory functionality
+version: 1.0.0
+vendor: rapid7
+support: rapid7
+status: []
+tags: [TEM, endpoint, patch]
+hub_tags:
+  use_cases: [application_management, threat_detection_and_response, asset_inventory, remediation_management]
+  keywords: [TEM, endpoint, patch]
+  features: []
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/ivanti_security_controls
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://www.ivanti.com/
+enable_cache: false
+
+connection:
+  host:
+    title: Host
+    description: Enter the hostname
+    type: string
+    required: true
```

Comment:
New `example` keys for the connection inputs.

Reply:
For the credential type, it will be the username/password combo. 

`example: '{"username":"user1", "password":"mypassword"}'`